README.md

```diff
@@ -46,10 +46,10 @@ yarn start
 
 ### Terminal 3 - Node
 
-Follow up installation instructions from the [Polkadot repo](https://github.com/paritytech/polkadot)
```

Comment:
I think we want to stick with polkadot as the example; devs usually understand they can run any substrate chain.